tests/view_tests/tests/test_debug.py

```diff
@@ -596,72 +580,89 @@ class ExceptionReporterFilterTests(TestCase, ExceptionReportTestMixin):
     Ensure that sensitive information can be filtered out of error reports.
     Refs #14614.
     """
-    rf = RequestFactory()
+    client_class = SensitiveTestClient
+
+    def setUp(self):
+        logger = logging.getLogger('django.request')
+        log_handler = logger.handlers[0]
```

Comment:
I've backported `assertLogs` in #3467. Perhaps we can merge `assertLogs` part of the PR first.